.github/workflows/deploy.yml

```diff
@@ -74,6 +89,21 @@ jobs:
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+      
+      - name: Deploy Course Preview to Netlify
+        uses: nwtgck/actions-netlify@v1.1
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          publish-dir: '../ocw-course/public'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          deploy-message: "Deploy from GitHub Actions"
+          enable-pull-request-comment: true
+          enable-commit-comment: false
+          overwrites-pull-request-comment: true
```

Comment:
I'm not sure if there's a way to get multiple "Netlify deploy" links to show up in the checks section.  The `nwtgck/actions-netlify` library seems to only support the one.  However, if we are going to do multiple Netlify deploys per PR we should try setting `overwrites-pull-request-comment` to `false` so a comment is written in for each deploy.  Currently, you might notice that since the course build / deployment runs second, it overwrites the comment about the `www` build.  Ithink setting this to `false` should give a unique comment for each deploy on the PR.